channels-rke2.yaml

```diff
@@ -1111,4 +1111,54 @@ releases:
         repo: rancher-rke2-charts
         version: 2.6.2-rancher200
     featureVersions: *featureVersions-v1
+  - version: v1.24.14-rc2+rke2r1
+    minChannelServerVersion: v2.6.7-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v1-24-2-rke2r1
+    agentArgs: *agentArgs-v1-24-2-rke2r1
+    charts: &charts-v1-24-13-rke2r1
+      <<: *charts-v1-24-11-rke2r1
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.25.1-build2023051200
+      rke2-cilium:
+        repo: rancher-rke2-charts
+        version: 1.13.200
+      harvester-csi-driver:
+        repo: rancher-rke2-charts
+        version: 0.1.1600
+      rke2-multus:
+        repo: rancher-rke2-charts
+        version: v3.9.3-build2023010902
+      rke2-calico:
+        repo: rancher-rke2-charts
+        version: v3.25.002
+      rke2-calico-crd:
+        repo: rancher-rke2-charts
+        version: v3.25.002
+      rke2-ingress-nginx:
+        repo: rancher-rke2-charts
+        version: 4.5.201
+      rke2-coredns:
+        repo: rancher-rke2-charts
+        version: 1.19.402
+      rancher-vsphere-cpi:
+        repo: rancher-rke2-charts
+        version: 1.5.100
+      rancher-vsphere-csi:
+        repo: rancher-rke2-charts
+        version: 3.0.1-rancher100
+      rke2-metrics-server:
+        repo: rancher-rke2-charts
+        version: 2.11.100-build2023051508
+      rke2-snapshot-controller:
+        repo: rancher-rke2-charts
+        version: 1.7.202
+      rke2-snapshot-controller-crd:
+        repo: rancher-rke2-charts
+        version: 1.7.202
+      rke2-snapshot-validation-webhook:
+        repo: rancher-rke2-charts
+        version: 1.7.101
```

Comment:
sorry this is my fault; I wasn't clear in our DM -- these don't exist in 1.24 so you can remove them!